### PR TITLE
Health records changes

### PR DESCRIPTION
**Pull request by iulianpascalau**
- added a new flag to set a custom value when the health records start triggering pprof files
- removed the custom changes whenever an import-db process starts

Testing scenario: start a testnet, take a node, stop it, and start it with these 2 custom flags: `-use-health-service -memory-usage-to-create-profiles 1073741824` which translates to: create health records, but alter the value to 1GB.
grep the log to the following string: `setting a new value for the memoryUsageToCreateProfiles option` and you should be able to check that the new value was set correctly (the default one from the config is ~2.25GB)
Also, check that health records are to be created in the `health-records` directory and contains files with the filenames in the following format: `mem__20220621124307__1_xx_GB.pprof`

### CODE REVIEW

File: cmd/node/flags.go [hunk 1]
```diff
@@ -499,14 +510,6 @@ func processConfigImportDBMode(log logger.Logger, configs *config.Configs) error
 	p2pConfigs.Node.ThresholdMinConnectedPeers = 0
 	p2pConfigs.KadDhtPeerDiscovery.Enabled = false
 
-	defaultSecondsToCheckHealth := 300                   // 5minutes
-	defaultDiagnoseMemoryLimit := 6 * 1024 * 1024 * 1024 // 6GB
-
-	generalConfigs.Health.IntervalDiagnoseComponentsDeeplyInSeconds = defaultSecondsToCheckHealth
-	generalConfigs.Health.IntervalDiagnoseComponentsInSeconds = defaultSecondsToCheckHealth
-	generalConfigs.Health.IntervalVerifyMemoryInSeconds = defaultSecondsToCheckHealth
```
Comment (ssd04) [inline cmd/node/flags.go:507]: how about these? it looks like there are not being set anymore

**Review by andreibancioiu — APPROVED** [review]

File: cmd/node/flags.go (hunk 1 above)
Comment (andreibancioiu) [reply cmd/node/flags.go:507]: Normally, they default to what it's written in `config.toml` (no CLI flags for them).

Comment (ssd04) [reply cmd/node/flags.go:507]: ah, yes :+1:

**Review by ssd04 — APPROVED** [review]

**Review by gabi-vuls — APPROVED** [review]
@@                            Log scanner                                  @@
##                            health-records-changes                       ##
================================================================================
- Known Warnings              8
- New Warnings                0
- Known Errors                0
- New Errors                  0
- Panics                      0
================================================================================

**Pull request merged**